portfolio.js

```diff
@@ -22,3 +22,244 @@ for (let i = 0; i < navLinks.length; i += 1) {
+
+  const popup = document.createElement('div');
+  popup.classList.add('popup');
+
+  const popupContent = document.createElement('div');
+  popupContent.classList.add('popup-content');
+
+  const closePopup = document.createElement('p');
+  closePopup.classList.add('close-popup');
+  closePopup.innerHTML = '<i class="fa fa-times" aria-hidden="true"></i>';
+  popupContent.appendChild(closePopup);
+
+  const popupImage = document.createElement('div');
+  popupImage.classList.add('popup-image');
+  popupContent.appendChild(popupImage);
+
+  const popupText = document.createElement('div');
+  popupText.classList.add('popup-text');
+
+  const popupHeader = document.createElement('h2');
+  popupHeader.classList.add('popup-header');
+  popupHeader.textContent = projectsDetails[1].popup_title;
+  popupText.appendChild(popupHeader);
+
+  const technologies = document.createElement('ul');
+  technologies.classList.add('technologies');
+  for (let j = 0; j < projectsDetails[i].mobile_popup_technologies.length; j += 1) {
+    const list = document.createElement('li');
+    list.textContent = projectsDetails[i].mobile_popup_technologies[j];
+    technologies.appendChild(list);
+  }
+  popupText.appendChild(technologies);
+
+  const technologiesDesktop = document.createElement('ul');
+  technologiesDesktop.classList.add('technologies-desktop');
+  for (let j = 0; j < projectsDetails[i].desktop_popup_technologies.length; j += 1) {
+    const lang = document.createElement('li');
+    lang.textContent = projectsDetails[i].desktop_popup_technologies[j];
+    technologiesDesktop.appendChild(lang);
+  }
+  popupText.appendChild(technologiesDesktop);
+
+  const popupDesc = document.createElement('p');
+  popupDesc.textContent = projectsDetails[i].long_description;
+  popupText.appendChild(popupDesc);
+
+  const popupButtons = document.createElement('div');
+  popupButtons.classList.add('popup-buttons');
+
+  const button1 = document.createElement('button');
+  button1.type = 'button';
+  button1.innerHTML = '<a href="https://ceemos96.github.io/Portfolio-setup-and-mobile-version-skeleton/" target="_blank"> See Live <i class="fa fa-link" ></i>';
+  popupButtons.appendChild(button1);
+
+  const button2 = document.createElement('button');
+  button2.type = 'button';
+  button2.innerHTML = '<a href="https://github.com/Ceemos96/Portfolio-setup-and-mobile-version-skeleton" target="_blank"> See Source  <i class="fab fa-github"></i></a>';
+  popupButtons.appendChild(button2);
+
+  popupText.appendChild(popupButtons);
+
+  popupContent.appendChild(popupText);
+
+  popup.appendChild(popupContent);
+
+  popupWrapper.appendChild(popup);
+
+  projectsContainer.appendChild(popupWrapper);
+
+  // function openModal() {
+  //   document.querySelector('.popupWrapper').style.display = 'block';
+  //   console.log('yay')
+  // }
+
+  // function closeModal() {
+  //   document.querySelector('.popupWrapper').style.display = 'none';
+  //   console.log('yay')
+  // }
+
```

Comment:
**[REQUIRED CHANGES]**

- [ ] Please, kindly remove this unused commented code here also, it makes your codebase not clean and makes you not a professional.